snuba/query/project_extension.py

```diff
@@ -107,14 +107,22 @@ class ProjectWithGroupsProcessor(ProjectExtensionProcessor):
     2. Taking into consideration groups that should be excluded (groups are excluded because of replacement).
     """
 
+    def __init__(self, project_column: str, replacer_state_name: ReplacerState) -> None:
+        super().__init__(project_column)
+        # This is used to disambiguate datasets in Redis when reading query flags.
+        # If not present, infer the table name from the query.
```

Comment:
This comment doesn't seem accurate

Reply:
right, it came from the previous version. Will rephrase